### PR TITLE
Create Changelog for v1.11.1 release

### PR DESCRIPTION
**Pull request by zeripath**

### CODE REVIEW

**Comment by zeripath** [comment]
I've included #10285

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ This changelog goes through all the changes that have been made in each release
...
+## [1.11.1](https://github.com/go-gitea/gitea/releases/tag/v1.11.1) - 2020-02-15
+
+* BUGFIXES
+  * Repo name added to automatically generated commit message when merging (#9997) (#10285)
+  * Fix Workerpool deadlock (#10283) (#10284)
+  * Divide GetIssueStats query in smaller chunks (#10176) (#10282)
+  * Fix reply on code review (#10257)
+  * Stop hanging issue indexer initialisation from preventing shutdown (#10243) (#10249)
+  * Fix filter label emoji width (#10241) (#10244)
+  * Backport - Fix issue sidebar menus having an infinite height (#10240)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Fix issue sidebar menus having an infinite height (#10239) (#10240)
```

**Review by jolheiser — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Review by techknowlogick — DISMISSED** [review]
blocking until #10292 backport

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ This changelog goes through all the changes that have been made in each release
...
+* BUGFIXES
+  * Repo name added to automatically generated commit message when merging (#9997) (#10285)
+  * Fix Workerpool deadlock (#10283) (#10284)
+  * Divide GetIssueStats query in smaller chunks (#10176) (#10282)
+  * Fix reply on code review (#10257)
+  * Stop hanging issue indexer initialisation from preventing shutdown (#10243) (#10249)
+  * Fix filter label emoji width (#10241) (#10244)
+  * Fix issue sidebar menus having an infinite height (#10239) (#10240)
+  * Fix commit between two commits calculation if there is only last commit (#10225) (#10226)
+  * Only check for conflicts/merging if the PR has not been merged in the interim (#10132) (#10206)
```
Comment (zeripath) [inline CHANGELOG.md:18]:
```suggestion
  * Only check for conflicts/merging if the PR has not been merged in the interim (#10132) (#10206)
  * Blacklist manifest.json & milestones user (#10292) (#10293)
```

**Review by 6543 — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Comment by 6543** [comment]
ping lgtm

**Pull request merged**